app/src/test/java/com/nervousfish/nervousfish/data_objects/Ed25519KeyTest.java

```diff
@@ -69,7 +69,7 @@ public void testGetKeyReturnsNonEmptyString() {
     @Test
     public void testGetTypeReturnsSimple() {
         IKey key = new Ed25519Key("Webserver", "foobar");
-        assertEquals("simple", key.getType());
+        assertEquals("ed25519", key.getType());
```

Comment:
Can you use ConstantKeywords.ED25519_KEY? That's type safe.
Can you also remove SECURITY_CODE defined in ConstantKeywords, because that appears to be unused